opba-db/src/test/java/de/adorsys/opba/db/CsvConvertTest.java

```diff
@@ -0,0 +1,55 @@
+package de.adorsys.opba.db;
+
+import com.google.common.io.Files;
+import com.google.common.io.Resources;
+import lombok.SneakyThrows;
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.Test;
+
+import java.io.File;
+import java.util.List;
+
+import static java.nio.charset.StandardCharsets.UTF_8;
+
+@Disabled
```

Comment:
It should be enabled/disabled based on some variable/property

Reply:
Test is enabled by the environment variable